### DISCUSSION TITLE
integrating webgazer to jsPsych

### DISCUSSION THREAD

**Discussion by mecui22** [jspsych/jsPsych#1330] [Q&A]
Hello, has anyone successfully integrated webgazer into their jsPsych script? alternatively, any suggestion for eye-tracking tools?

webgazer: https://webgazer.cs.brown.edu/

many thx!

**Comment by jodeleeuw** [comment]
We're working on a general extension to jsPsych's framework that would enable things like eye tracking via webgazer (#1123). I'm planning on using this when I teach my spring research methods course so it'll be ready by the end of January.

I also happened to come across [this preprint](https://psyarxiv.com/qhme6/) this morning, which describes a jspsych+webgazer implementation and test.

**Reply by mecui22** [reply]
thanks! excited about the new extension!
